source/includes/stackpath/_scripts.md

```diff
@@ -217,7 +217,7 @@ Edit a script in a given [environment](#administration-environments) for a given
 ```shell
 curl -X PUT \
    -H "MC-Api-Key: your_api_key" \
-   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/scripts?siteId=:siteId"
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/scripts/dd207010-4570-43ee-9ff2-5421d2306b41?siteId=0a57855b-26d8-4e8f-8b77-429997c7c5fb"
```

Comment:
you dont need to provide the id itself, only `:id` and `siteId` is good enough. This is what we do for the other endpoints no ?

Reply:
I think its not consistent. You could take a look at http://localhost:4567/#stackpath-retrieve-a-script and http://localhost:4567/#stackpath-retrieve-a-workload

Reply:
I see, all good then